src/components/05-alerts/alert.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * JSX Alert Component Attributes
+ * @function Alert
+ * @param {Object} config - The configuration object for alert attributes.
+ * @property {string} config.class - The classes inherited from the USWDS CSS files. The default class is `usa-alert` and additions can be appended.
+ * @property {string} [config.headerText] - The text to be displayed as the header for the alert. (Optional)
+ * @property {string} config.innerText - The actual alert message text to be displayed in the body of the Alert element.
+ * @example
+ * const config = {
+ *      class: "usa-alert--success",
+ *      headerText: "Success status"
+ *      innerText: "Your submission was successfull! You should recieve an email from us shortly."
+ * }
+ * @example -
+ *  const config = {
+ *      class: "usa-alert--info usa-alert--slim",
+ *      headerText: null
+ *      innerText: "Note: Some aspects of the site are a work in progress and may have some bugs. We appreciate your patience."
+ * }
+ * @returns {HTMLDivElement} The HTML Alert element
+ */
+
+exports.Alert = function (config) {
+  return (
+    <div class={`usa-alert ${config.class}`}>
+      <div class="usa-alert__body">
```

Comment:
non-blocker, but is there an simple way to allow extra body, heading and text classes to be passed in?

Reply:
I guess I could just add extra properties in the config object for those, then do the same thing I do on line 25.